tests/integrations/test_identity.py

```diff
@@ -0,0 +1,113 @@
+import json
+import time
+from pathlib import Path
+from typing import Generator
+
+import jwt
+import pytest
+
+from cryptography.hazmat.primitives.asymmetric import rsa
```

Comment:
I don't think `cryptography` is already an explicit dep in setup.py. I guess it works because it's a transitive dependency of PyJWT or something. But if we're importing it like that, I think we should make that an explicit dependency

Reply:
Ok I didn't consider this was just a test dependency. This makes it indeed much less an important breaking point and I'm fine with leaving it implicit, which has advantages too as you noted